test/e2e/testing-manifests/storage-csi/mock/csi-mock-proxy.yaml

```diff
@@ -73,7 +73,7 @@ spec:
         # test for directories or create them. It needs additional privileges
         # for that.
         - name: busybox
-          image: k8s.gcr.io/busybox
+          image: k8s.gcr.io/e2e-test-images/busybox:1.29-1
```

Comment:
Yay, we caught a bug!

Reply:
now do we need a test for a test util LOL (kidding)

Reply:
lol, I was considering that :stuck_out_tongue_winking_eye: